mscore/editstyle.ui

```diff
@@ -7661,7 +7661,7 @@
         </item>
        </layout>
       </widget>
-      <widget class="QWidget" name="page">
+      <widget class="QWidget" name="PageStaffText">
```

Comment:
This change would not cherry-pick to a 2.x branch, nor is needed there. Actually it is even really related to the topic of this PR...